packages/hs-react-ui/package.json

```diff
@@ -118,7 +122,47 @@
     "clean": "rm -rf ./dist/*",
     "webpack": "webpack --config webpack.config.js",
     "dts": "tsc --declaration --outDir dist/types/ --emitDeclarationOnly --declarationMap --allowJs false --checkJs false",
-    "test": "jest --collectCoverage=true"
+    "pre-push": "yarn run lint-ci && yarn run test && yarn run clean && yarn run webpack && yarn run dts",
+    "test": "rm -rf ./coverage/* && jest --collectCoverage=true",
+    "semver-dryrun": "npx semantic-release -d --debug",
+    "semver-release": "npx semantic-release",
+    "commit": "npx git-cz"
+  },
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/git",
+    [
+      "@semantic-release/npm",
+      {
+        "npmPublish": true
+      }
+    ]
+  ],
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "release": {
+    "branches": [
+      "master",
+      "v1-release",
+      "next"
+    ],
+    "prepare": [
+      "@semantic-release/changelog",
+      {
+        "path": "@semantic-release/git",
+        "assets": [
+          "package.json",
+          "../../docs/**",
+          "./coverage/**",
+          "CHANGELOG.md"
+        ],
+        "message": "chore(release): ${nextRelease.version} [skip ci]nn${nextRelease.notes}"
+      }
+    ]
```

Comment:
This is going to make the versions auto increment without us needing to do that, is that right?

Reply:
Yes, it will! Semantic release will auto version based on commit messages